src/Extensions/Diagnostics.HealthChecks/Internal/HttpEndpointHealthCheck.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	internal class HttpEndpointHealthCheck : AbstractHealthCheck
+	{
+		private const string Host = "localhost";
+
+		public static string HttpClientLogicalName { get; } = "HttpEndpointHealthCheckHttpClient";
+
+		internal HttpHealthCheckParameters Parameters { get; } // done internal for unit tests, please don't use it outside of class
```

Comment:
"internal only to be used for unit tests" reads better

Reply:
Thank you, updated